code/game/objects/items/inspector.dm

```diff
@@ -37,7 +37,7 @@
 
 // Clean up the cell on destroy
 /obj/item/clothing/suit/space/Destroy()
-	if(cell)
+	if(isatom(cell))
```

Comment:
Okay, get ready for this.

`cell` starts out as a type.
In `Initialize`, the cell is replaced by an atom version of itself.
But because we're creating stuff in the assets SS (which was moved *before* atoms, to be as responsive as possible), *and* destroying them, that means we're destroying things before Initialize is called.
There's a few ways around this, such as deferring deletion, but for reasons I don't remember the best solution ended up just being to patch around the few Destroy() calls before Initialize.